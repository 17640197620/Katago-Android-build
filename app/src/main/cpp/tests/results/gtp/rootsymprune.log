: GTP Engine starting...
: KataGo v1.10.0
: Using TrompTaylor rules initially, unless GTP/GUI overrides this
: Using 1 CPU thread(s) for search
: nnRandSeed0 = forTesting
: After dedups: nnModelFile0 = tests/models/g170-b6c96-s175395328-d26788732.bin.gz useFP16 false useNHWC false
: Initializing neural net buffer to be size 19 * 19 exactly
: Cuda backend thread 0: Found GPU Tesla V100-SXM2-16GB memory 16945512448 compute capability major 7 minor 0
: Cuda backend thread 0: Model version 8 useFP16 = false useNHWC = false
: Cuda backend thread 0: Model name: g170-b6c96-s175395328-d26788732
: Loaded neural net with nnXLen 19 nnYLen 19
: Initializing board with boardXSize 19 boardYSize 19
: Loaded config configs/gtp_example.cfg and/or command-line and query overrides
: Loaded model tests/models/g170-b6c96-s175395328-d26788732.bin.gz
: Config override: cudaUseFP16 = false
: Config override: cudaUseNHWC = false
: Config override: forDeterministicTesting = true
: Config override: logDir = 
: Config override: logFile = tests/results/gtp/rootsymprune.log
: Config override: logTimeStamp = false
: Config override: maxPlayouts = 10000
: Config override: maxVisits = 100
: Config override: nnRandSeed = forTesting
: Config override: nnRandomize = false
: Config override: numSearchThreads = 1
: Config override: openclUseFP16 = false
: Config override: rootSymmetryPruning = true
: Config override: searchRandSeed = forTesting
: Config override: trtUseFP16 = false
: Model name: g170-b6c96-s175395328-d26788732
: GTP ready, beginning main protocol loop
: Controller: genmove_debug b
: MoveNum: 0 HASH: CDCBC1F514D7E680FACD226074256633
   A B C D E F G H J K L M N O P Q R S T
19 . . . . . . . . . . . . . . . . . . .
18 . . . . . . . . . . . . . . . . . . .
17 . . . . . . . . . . . . . . . . . . .
16 . . . . . . . . . . . . . . . . @ . .
15 . . . . . . . . . . . . . . . . . . .
14 . . . . . . . . . . . . . . . . . . .
13 . . . . . . . . . . . . . . . . . . .
12 . . . . . . . . . . . . . . . . . . .
11 . . . . . . . . . . . . . . . . . . .
10 . . . . . . . . . . . . . . . . . . .
 9 . . . . . . . . . . . . . . . . . . .
 8 . . . . . . . . . . . . . . . . . . .
 7 . . . . . . . . . . . . . . . . . . .
 6 . . . . . . . . . . . . . . . . . . .
 5 . . . . . . . . . . . . . . . . . . .
 4 . . . . . . . . . . . . . . . . . . .
 3 . . . . . . . . . . . . . . . . . . .
 2 . . . . . . . . . . . . . . . . . . .
 1 . . . . . . . . . . . . . . . . . . .

koPOSITIONALscoreAREAtaxNONEsui1komi7.5
Time taken: ###
Root visits: 100
New playouts: 100
NN rows: 80
NN batches: 80
NN avg batch size: 1
PV: R16 Q3 C16 D3 E16
Tree:
: T   0.20c W  -0.16c S   0.37c ( +0.2 L  +0.1) N     100  --  R16 Q3 C16 D3 E16
---Black(^)---
R16 : T   0.42c W   0.03c S   0.40c ( +0.3 L  +0.2) LCB   -0.61c P  7.34% WF  91.1 PSV      91 N      91  --  R16 Q3 C16 D3 E16
Q16 : T  -1.17c W  -1.43c S   0.13c ( -0.1 L  -0.1) LCB  -19.00c P  3.90% WF   5.9 PSV       6 N       6  --  Q16 D3 D17 Q3
R17 : T  -2.64c W  -2.45c S  -0.19c ( -0.6 L  -0.6) LCB -280.00c P  1.71% WF   2.0 PSV       1 N       2  --  R17 D3

: = R16
: Controller: genmove_debug w
: MoveNum: 1 HASH: 48315B528346F7FF166D01D0584718A4
   A B C D E F G H J K L M N O P Q R S T
19 . . . . . . . . . . . . . . . . . . .
18 . . . . . . . . . . . . . . . . . . .
17 . . . . . . . . . . . . . . . . . . .
16 . . . . . . . . . . . . . . . . X1. .
15 . . . . . . . . . . . . . . . . . . .
14 . . . . . . . . . . . . . . . . . . .
13 . . . . . . . . . . . . . . . . . . .
12 . . . . . . . . . . . . . . . . . . .
11 . . . . . . . . . . . . . . . . . . .
10 . . . . . . . . . . . . . . . . . . .
 9 . . . . . . . . . . . . . . . . . . .
 8 . . . . . . . . . . . . . . . . . . .
 7 . . . . . . . . . . . . . . . . . . .
 6 . . . . . . . . . . . . . . . . . . .
 5 . . . . . . . . . . . . . . . . . . .
 4 . . . . . . . . . . . . . . . . @ . .
 3 . . . . . . . . . . . . . . . . . . .
 2 . . . . . . . . . . . . . . . . . . .
 1 . . . . . . . . . . . . . . . . . . .

koPOSITIONALscoreAREAtaxNONEsui1komi7.5
Time taken: ###
Root visits: 100
New playouts: 9
NN rows: 10
NN batches: 10
NN avg batch size: 1
PV: Q3 C16 D3 E16
Tree:
: T  -0.20c W  -0.08c S  -0.07c ( -0.3 L  -0.2) N     100  --  Q3 C16 D3 E16
---White(^)---
Q3  : T   0.35c W   0.36c S  -0.03c ( -0.2 L  -0.1) LCB   -1.99c P 12.02% WF  18.1 PSV      27 N      18  --  Q3 C16 D3 E16
D3  : T  -0.12c W   0.11c S  -0.08c ( -0.3 L  -0.1) LCB   -2.75c P 13.86% WF  18.0 PSV      18 N      18  --  D3 Q3 D17 Q5
C16 : T   0.75c W   0.42c S  -0.01c ( -0.2 L  -0.1) LCB   -3.22c P  8.75% WF  15.1 PSV      15 N      15  --  C16 D3 Q3 D5
R4  : T  -0.65c W  -0.65c S  -0.16c ( -0.4 L  -0.2) LCB   -4.85c P 11.91% WF  11.9 PSV      12 N      12  --  R4 D3 D17 D5
D17 : T  -0.35c W  -0.41c S  -0.11c ( -0.3 L  -0.3) LCB   -6.06c P 10.44% WF  11.0 PSV      11 N      11  --  D17 D3 Q3 D5
C4  : T  -0.46c W  -0.39c S  -0.11c ( -0.3 L  -0.2) LCB   -7.56c P  9.17% WF  10.0 PSV      10 N      10  --  C4 Q3 D17 Q5
Q16 : T  -1.22c W   0.13c S   0.08c ( -0.1 L  -0.4) LCB  -30.47c P  7.32% WF   6.0 PSV       6 N       6  --  Q16 Q17 P17 R17
D4  : T  -0.95c W  -0.56c S  -0.19c ( -0.5 L  -0.5) LCB -231.16c P  5.92% WF   3.0 PSV       3 N       3  --  D4 C16 Q3
D16 : T  -1.72c W  -1.03c S  -0.25c ( -0.5 L  -0.6) LCB -218.59c P  5.73% WF   3.0 PSV       3 N       3  --  D16 D3 Q3
Q4  : T  -0.85c W  -0.20c S  -0.16c ( -0.4 L  -0.5) LCB -159.77c P  5.07% WF   3.0 PSV       3 N       3  --  Q4 D3 D17

: = R4
: Controller: genmove_debug b
: MoveNum: 2 HASH: BB0B7C9F995E20305FE59452D771F13B
   A B C D E F G H J K L M N O P Q R S T
19 . . . . . . . . . . . . . . . . . . .
18 . . . . . . . . . . . . . . . . . . .
17 . . . . . . . . . . . . . . . . . . .
16 . . @ . . . . . . . . . . . . . X1. .
15 . . . . . . . . . . . . . . . . . . .
14 . . . . . . . . . . . . . . . . . . .
13 . . . . . . . . . . . . . . . . . . .
12 . . . . . . . . . . . . . . . . . . .
11 . . . . . . . . . . . . . . . . . . .
10 . . . . . . . . . . . . . . . . . . .
 9 . . . . . . . . . . . . . . . . . . .
 8 . . . . . . . . . . . . . . . . . . .
 7 . . . . . . . . . . . . . . . . . . .
 6 . . . . . . . . . . . . . . . . . . .
 5 . . . . . . . . . . . . . . . . . . .
 4 . . . . . . . . . . . . . . . . O2. .
 3 . . . . . . . . . . . . . . . . . . .
 2 . . . . . . . . . . . . . . . . . . .
 1 . . . . . . . . . . . . . . . . . . .

koPOSITIONALscoreAREAtaxNONEsui1komi7.5
Time taken: ###
Root visits: 100
New playouts: 88
NN rows: 77
NN batches: 77
NN avg batch size: 1
PV: C16 C4 P16 E4 E16
Tree:
: T   0.78c W   0.60c S   0.12c ( +0.4 L  +0.3) N     100  --  C16 C4 P16 E4 E16
---Black(^)---
C16 : T   1.47c W   1.09c S   0.22c ( +0.6 L  +0.4) LCB    0.19c P 18.58% WF  29.2 PSV      48 N      29  --  C16 C4 P16 E4 E16
C4  : T   1.06c W   0.79c S   0.12c ( +0.4 L  +0.3) LCB   -0.10c P 18.70% WF  23.0 PSV      23 N      23  --  C4 D17 E4 P4 P16
D3  : T   0.83c W   0.72c S   0.08c ( +0.4 L  +0.3) LCB   -0.56c P 18.56% WF  20.0 PSV      20 N      20  --  D3 D17 D5 P4 P16
D17 : T   0.58c W   0.32c S   0.12c ( +0.4 L  +0.3) LCB   -2.35c P 17.96% WF  20.9 PSV      19 N      21  --  D17 D3 P16 D5 D15
Q4  : T  -3.52c W  -1.64c S  -0.53c ( -0.5 L  -0.3) LCB -280.00c P  5.63% WF   2.0 PSV       2 N       2  --  Q4 Q3
D16 : T  -2.37c W  -1.37c S  -0.18c ( +0.0 L  -0.1) LCB -280.00c P  4.73% WF   2.0 PSV       2 N       2  --  D16 D3
D4  : T  -2.11c W  -1.13c S  -0.13c ( +0.1 L  +0.1) LCB -280.00c P  4.18% WF   2.0 PSV       2 N       2  --  D4 D17

: = C16
: GPU -1 finishing, processed 167 rows 167 batches
: All cleaned up, quitting
