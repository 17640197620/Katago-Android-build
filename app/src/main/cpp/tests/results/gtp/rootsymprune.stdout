= R16

= R4

= C16

